contracts/crowdsale/validation/WhitelistCrowdsale.sol

```diff
@@ -4,10 +4,10 @@ import "../../access/roles/WhitelistedRole.sol";
 
 
 /**
- * @title WhitelistedCrowdsale
+ * @title WhitelistCrowdsale
  * @dev Crowdsale in which only whitelisted users can contribute.
  */
-contract WhitelistedCrowdsale is WhitelistedRole, Crowdsale {
+contract WhitelistCrowdsale is WhitelistedRole, Crowdsale {
```

Comment:
We can't remove `WhitelistedCrowdsale` because it's a breaking change. I wouldn't deprecate it for this simple name change. I don't think it's worth it.

Reply:
Not a breaking change, `WhitelistedCrowdsale` was removed prior to the 2.0 release: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1291/files#diff-3e88d60d6ebc203375356e8eeb944a56

This is a new feature.